MANIFEST.in

```diff
@@ -4,6 +4,7 @@ include docs/conf.py
 include docs/test_server.ini
 include regen-docs
 include tox.ini
+include LICENSE
```

Comment:
There is no LICENSE file in the git repo so I think this is just a no-op.

Reply:
But anyway.  I think this change won't fully solve #101 because even with this it won't be possible to automatically find the real and full license wording.

Reply:
I'm not sure I follow because it seems to be working as expected:

After:

1. added that to MANIFEST.in
2. making LICENSE a symlink to docs/license.txt

then the PKG-INFO has: `License-File: LICENSE`

and the created tarball includes the text of the license in the included LICENSE file

The created wheel does _not_ but I wouldn't expect it to?

Reply:
Ah, I missed the `LICENSE` is a symlink.  I thought it is a regular file.  The github GUI is far from ideal in this.  I'm sorry for confusion and thank you for patience and the fix.